engine/src/vsfilesystem.cpp

```diff
@@ -707,14 +725,22 @@ void LoadConfig( string subdir )
         BOOST_LOG_TRIVIAL(info) << boost::format("DATADIR - Found a datadir in config, using : %1%") % game_options.datadir;
         datadir = game_options.datadir;
     } else {
-        BOOST_LOG_TRIVIAL(info) << boost::format("DATADIR - No datadir specified in config file, using : %1%") % datadir;
+        if (true == legacy_mode) {
+            BOOST_LOG_TRIVIAL(info) << boost::format("DATADIR - No datadir specified in config file, using : %1%") % datadir;
+        } else {
+            BOOST_LOG_TRIVIAL(fatal) << boost::format("DATADIR - No datadir specified in config file");
```

Comment:
Just to make sure we're on the same page here, `BOOST_LOG_TRIVIAL(fatal)` does not, by itself, exit the program. Call `VSExit` to do that.

Reply:
Thanks! done